src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java

```diff
@@ -83,7 +83,7 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
 
             asylumCase.write(IS_HOME_OFFICE_INTEGRATION_ENABLED, YesOrNo.YES);
 
-            if (HandlerUtils.isAppellantInDetention(asylumCase) || HandlerUtils.isAgeAssessmentAppeal(asylumCase)) {
```

Comment:
Will this break functionality for non EJP cases - Appeal in detention?

Reply:
yep you're right that there is actually a non EJP detained journey. I've updated and tested to make sure that scenario is still supported, thanks.